docs/src/bpmn-workflows/events.md

```diff
@@ -1,7 +1,49 @@
 # Events
 
-Currently supported elements:
+![workflow](/bpmn-workflows/events.png)
+
+Events in BPMN can be __thrown__ (i.e. sent), or __caught__ (i.e. received),
+respectively referred to as *throw* or *catch* events,
+e.g. `message throw event`, `timer catch event`.
+
+Additionally, a distinction is made between start, intermediate, and end events:
+
+* Start events (catch events, as they can only react to something) are used to denote
+the beginning of a process or sub-process.
+* End events (throw events, as they indicate something has happened) are used to denote the
+end of a particular sequence flow.
+* Intermediate events can be used to indicate that something has happened
+(i.e. intermediate throw events), or to wait and react to certain events
+(i.e. intermediate catch events).
+
+## Intermediate events
+
+Intermediate catch events can be inserted into your process in two different
+contexts: normal flow, or attached to an activity, and are called boundary events.
+
+### Normal flow
```

Comment:
I would not say that a boundary event is an intermediate event that is attached to an activity. I think this doesn't fit to BPMN well. 

```suggestion
## Intermediate Events
```